filters/filter_17_Chinese/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 224,
```

Comment:
is it still problematic to have `filterId=17`? even if it's obsolete filter replacement and not new filter adding

or the problems may happen exactly because of `filterId`?

Reply:
I am not sure how old AdGuard versions will handle complete filter list removal so it's safer to replace a filter list instead.